src/components/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import { Icon } from "../Icon/Icon";
+
+export const LinkButton = ({
+  href,
+  text,
+  variant = "primary",
+  className = "",
+}: {
+  href: string;
+  text: string;
+  variant?: "primary" | "secondary";
+  className?: string;
+}) => {
+  return (
+    <Button
+      asChild
+      variant={variant}
+      className={`w-full md:w-auto ${className}`}
+    >
+      <Link href={href}>
+        {text}
+        {variant === "secondary" && (
+          <Icon
+            id="expand"
+            className="transform -rotate-90 !w-[8px] !h-[8px]"
+          />
+        )}
```

Comment:
Instead of adding this text and icon on the secondary variant, add the prop `children` to it, doing this we make this component even more generic and reusable :)
```suggestion
        {children}
```